packaging/centos7/cloud.spec

```diff
@@ -54,7 +54,7 @@ intelligent IaaS cloud implementation.
 %package management
 Summary:   CloudStack management server UI
 Requires: java-1.8.0-openjdk
-Requires: python
+Requires: python3
```

Comment:
We probably want both python and python3 like the debian/control change @GabrielBrascher, please add a new Requires line

Reply:
@rhtyd ah sure. Updated it, thanks!